### ISSUE TITLE
Running GNN and CNN agent into one

### ISSUE THREAD

**Issue by pdx97** [ZhengyaoJiang/GTG#3]
Hi,

I am using your GTG paper for something I am working , I just wanted to know is there any way I can use the fully connected embeddings from the GNN agent and the fully connected embeddings from the CNN agent into one before passing it to the policy logits , because your implementation deals with GNN and CNN separately .

Thanks
